app/src/full/java/io/homeassistant/companion/android/sensors/AndroidAutoSensorManager.kt

```diff
@@ -79,14 +203,190 @@ class AndroidAutoSensorManager : SensorManager, Observer<Int> {
                 false to "Unknown($value)"
             }
         }
-        onSensorUpdated(
-            context,
-            androidAutoConnected,
-            connected,
-            androidAutoConnected.statelessIcon,
-            mapOf(
-                "connection_type" to typeString
+
+        if (connected && !alreadyConnected) {
+            startAppNotification()
+            alreadyConnected = true
+        } else if (!connected && alreadyConnected) {
+            alreadyConnected = false
+            connectList.forEach { connectList[it.key] = false }
+        }
+
+        if (isEnabled(context, androidAutoConnected)) {
+            onSensorUpdated(
+                context,
+                androidAutoConnected,
+                connected,
+                androidAutoConnected.statelessIcon,
+                mapOf(
+                    "connection_type" to typeString
+                )
             )
-        )
+        }
+    }
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    private fun startAppNotification() {
+        val manager = CarNotificationManager.from(context)
+
+        val channelID = "HA_AA_OPEN"
```

Comment:
This needs to be added to the internal channel list that we maintain.

https://github.com/home-assistant/android/blob/master/common/src/main/java/io/homeassistant/companion/android/common/util/AppNotifChannels.kt